lib/chef/knife/opc_org_user_remove.rb

```diff
@@ -36,12 +43,39 @@ def run
       end
 
       org = Chef::Org.new(@org_name)
+
+      if config[:force_remove_from_admins]
+        begin
+          org.remove_user_from_group("admins", username)
+          org.remove_user_from_group("billing-admins", username)
+        rescue Net::HTTPServerException => e
+          if e.response.code == "404"
+            ui.msg "User #{username} is not in the admins group for organization #{org_name}. Do not pass --force_remove_from_admins."
```

Comment:
Maybe we should continue without failing in this case?  The end-state is what the user said they desired.  That would make this easier to use in some automation or wrapped in some other tooling.  You may need, however, to seperate the admins and billing-admins removal so we always do both even when one failed with 404.